### PR TITLE
Upgrade to PHP8

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

**Comment by Baggerone** [comment]
This isn't complaining about https://github.com/silinternational/yii2-email-log-target
not being updated in any way.

Is that OK, or do we need to try to upgrade yii2-email-log-target as well somehow?

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ deps:
 	docker-compose run --rm cli composer install
 
 depsupdate:
-	docker-compose run --rm cli bash -c "composer update && composer show -Df json > versions.json"
+	docker-compose run --rm cli bash -c "git config --global --add safe.directory /data/vendor/silinternational/yii2-codeception && composer update && composer show -D > versions.json"
```
Comment (forevermatt) [inline Makefile]: It seems odd to see `git config` calls when merely trying to update composer dependencies. What necessitated this?

Comment (Baggerone) [reply Makefile]: I was getting something similar to this (below). Apparently, there''s a new security feature that docker containers mess with somehow.

https://stackoverflow.com/questions/71901632/fatal-error-unsafe-repository-home-repon-is-owned-by-someone-else

File: Makefile [hunk 2]
```diff
@@ -7,7 +7,7 @@ deps:
 	docker-compose run --rm cli composer install
 
 depsupdate:
-	docker-compose run --rm cli bash -c "composer update && composer show -Df json > versions.json"
+	docker-compose run --rm cli bash -c "git config --global --add safe.directory /data/vendor/silinternational/yii2-codeception && composer update && composer show -D > versions.txt"
```
Comment (forevermatt) [inline Makefile]: @Baggerone I don't see an error message when I run `make depsupdate` without the "safe.directory" command. Could you share what the error message was?

Comment (Baggerone) [reply Makefile]: @forevermatt  I don't get the error anymore. So, I've backed out that config add part.

**Review by forevermatt — APPROVED** [review]

**Pull request merged**